### PR TITLE
snap-confine is now on /usr/lib/snapd

### PR DESCRIPTION
**Pull request by drizzt**

### CODE REVIEW

File: debian/usr.bin.snap-confine [hunk 1]
```diff
@@ -106,7 +106,7 @@
     mount options=(rw rbind) /var/tmp/ -> /tmp/snap.rootfs_*/var/tmp/,
     mount options=(rw rbind) /run/ -> /tmp/snap.rootfs_*/run/,
     mount options=(rw rbind) /media/ -> /tmp/snap.rootfs_*/media/,
-    mount options=(rw rbind) /lib/modules/ -> /tmp/snap.rootfs_*/lib/modules/,
+    mount options=(rw rbind) {/usr,}/lib/modules/ -> /tmp/snap.rootfs_*/lib/modules/,
```
Comment (zyga) [inline debian/usr.bin.snap-confine:109]: This seems unrelated

Comment (drizzt) [inline debian/usr.bin.snap-confine:109]: On ArchLinux /lib is a symlink to /usr/bin and /bin, /sbin, /usr/sbin are symlinks to /usr/bin.
So to make apparmor "happy" in order to bind mount /lib/modules you need to supports /usr/lib/modules too
